resources/assets/coffee/react/beatmaps/search-panel.coffee

```diff
@@ -52,14 +52,18 @@ class Beatmaps.SearchPanel extends React.PureComponent
 
   renderGuest: =>
     div
-      className: 'osu-page-header osu-page-header--beatmapsets-header-guest'
+      className: 'beatmapsets-search'
       div
         className: 'osu-page-header__background'
         style:
           backgroundImage: "url(#{@props.background})"
-      h1
-        className: 'osu-page-header__title'
-        'Beatmaps'
+      div className: 'fancy-search fancy-search--beatmapsets js-user-link',
+        input
```

Comment:
You can tab into the box and type while not triggering the popup. Maybe disable the input box or trigger the popup on enter or something?